pkg/daemon/update.go

```diff
@@ -2048,8 +2046,31 @@ func (dn *Daemon) experimentalUpdateLayeredConfig() error {
 	} else if liveUpdatedEquivalentTo == desiredImage {
 		glog.Info("No need to update, live update is equivalent")
 	} else {
+		if err := dn.setWorking(); err != nil {
+			return fmt.Errorf("failed to set working: %w", err)
+		}
 
 		client := &RpmOstreeClient{}
+		diffFileSet, err := Diff(currentImage, desiredImage)
```

Comment:
I think we're going to want to reorder some of this to instead:
```
rpm-ostree rebase desiredImage
rpm-ostree status
diffFileSet := ostree diff status->checksum
if reboot not in calculatePostConfigChangeActionFromFiles(diffFileSet) {
  rpm-ostree ex apply-live desiredImage
}
```

Reply:
Once we figure that I can rework drain and performPostConfigChangeActions

Reply:
Yeah, that makes sense. You can torch as much of experimentalUpdateLayeredConfig as you want, I was just trying to get the pipeline working as fast as possible.